contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,155 @@
+<frontmatter>
+  title: Workflow Automation with GitHub Actions
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# GitHub Actions
+
+📝 [Alfred Yip](https://github.com/alyip98)
+
+🔎 [James Pang](https://github.com/jamessspanggg), [Tan Yuanhong](https://github.com/le0tan), [Tejas Bhuwania](https://github.com/Tejas2805)
+
+
+<box id="article-toc">
+
+* [What is GitHub Actions‎](#what-is-github-actions)
+* [Why GitHub Actions‎](#why-github-actions)
+* [How does GitHub Actions Work‎](#how-does-github-actions-work)
+* [How is GitHub Actions Being Used‎](#how-is-github-actions-being-used)
+* [How to Get Started with GitHub Actions‎](#how-to-get-started-with-github-actions)
+* [Where to Go From Here‎](#where-to-go-from-here)
+</box>
+
+## What is GitHub Actions
+<!--Explain how X fits into the big picture of SE. Describe it relative to topics the reader is likely to know.-->
+GitHub Actions is a workflow automation solution that is tightly integrated into the GitHub ecosystem. It is commonly used to provide end-to-end continuous integration and continuous deployment (CI/CD) directly in your repository. With its tight integration with GitHub's ecosystem of services, users have the potential to automate more than just CI/CD.
+
+---
+
+## Why GitHub Actions
+<!--Motivate the reader to learn about X. Describe benefits of X to make the reader interested in X. Try to give a balanced view of by mentioning also WHY NOT X i.e., mention both advantages and disadvantages.-->
+
+### 1. Easy to Use
+Setting up a workflow in a repository is easy and hassle-free. [Starter workflows](https://github.com/actions/starter-workflows) for popular languages are available and the built-in editor offers code completion and documentation at a glance.
+![](images/editor.png)
+
+### 2. Reusable Actions
+Actions are small, reusable units in a workflow. When setting up a workflow, we can reuse actions written by the community. These actions are available at the marketplace.
+
+
+![](images/marketplace.png)
+
+
+
+### 3. Cross OS Support
+GitHub Actions gives you the option to [run workflows](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources) on Ubuntu, Windows and macOS.
+
+### 4. Docker Support
+GitHub Actions provides the ability to run docker images for projects that use a dockerized environment.
+
+### 5. Powerful
+Integration with the GitHub ecosystem allows GitHub Actions to do more than just build/test/deploy. You can automate everything related to your GitHub project, including bug triaging, labeling, automated dependency updating etc.
```

Comment:
>Integration with the GitHub ecosystem allows GitHub Actions to do more than just build/test/deploy.

Something very similar was mentioned earlier in the article. Given this is a short chapter, try to avoid repetition.